.github/workflows/owasp-dependency-check-daily.yml

```diff
@@ -0,0 +1,48 @@
+# the benefit of this over renovate is that this also analyzes transitive dependencies
+# while renovate (at least currently) only analyzes top-level dependencies
+name: OWASP dependency check (daily)
+
+on:
+  schedule:
+    # daily at 1:30 UTC
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+jobs:
+  analyze:
+    runs-on: otel-linux-latest-8-cores
```

Comment:
it's hanging when I run on my fork (where I don't have large runners available)

I can't test this here until after merging

Reply:
nevermind, I think I figured it out